tormenta-avro/src/main/scala/com/twitter/tormenta/scheme/avro/specific/SpecificAvroScheme.scala

```diff
@@ -17,48 +17,21 @@
 package com.twitter.tormenta.scheme.avro.specific
 
 import org.apache.avro.Schema
-import com.twitter.tormenta.scheme.avro.AvroScheme
-import com.twitter.tormenta.scheme.Scheme
+import com.twitter.tormenta.scheme.avro.BufferableScheme
 import com.twitter.bijection.avro.SpecificAvroCodecs
 import com.twitter.bijection.Injection._
-import java.nio.ByteBuffer
 import org.apache.avro.specific.SpecificRecordBase
 
-/**
- * @author Mansur Ashraf
- * @since 9/14/13
- */
 object SpecificAvroScheme {
-  def apply[T <: SpecificRecordBase: Manifest] = new SpecificAvroScheme[T]
-}
-
-class SpecificAvroScheme[T <: SpecificRecordBase: Manifest] extends Scheme[T] with AvroScheme[T] {
-  def decode(bytes: ByteBuffer): TraversableOnce[T] = {
-    implicit val inj = SpecificAvroCodecs[T]
-    decodeRecord(bytes)
-  }
+  def apply[T <: SpecificRecordBase: Manifest] = BufferableScheme(SpecificAvroCodecs[T])
```

Comment:
can we see if `: Manifest` can be replaced with `: ClassTag` using the latest bijection? Manifest is deprecated and we really need to move off of it. Removing it everywhere would be nice.